x11-packages/arc-gtk-theme/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/jnsh/arc-theme/
+TERMUX_PKG_DESCRIPTION="A flat theme with transparent elements"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=20221218
+TERMUX_PKG_SRCURL=https://github.com/jnsh/arc-theme/releases/download/${TERMUX_PKG_VERSION}/arc-theme-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=264570cc90a13b88d181334fb325f99a6ddc2a45392aa9ed7dba312aea042bb0
+TERMUX_PKG_DEPENDS="sassc"
```

Comment:
```suggestion
TERMUX_PKG_BUILD_DEPENDS="sassc"
```
`sassc` appears to be a make dependency only.
https://gitlab.archlinux.org/archlinux/packaging/packages/arc-gtk-theme/-/blob/main/PKGBUILD?ref_type=heads#L21

Reply:
fixed. thanks!